src/coverlet.collector/build/netstandard2.0/coverlet.collector.targets

```diff
@@ -26,7 +26,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <PropertyGroup>
     <_CoverletSdkNETCoreSdkVersion>$(NETCoreSdkVersion)</_CoverletSdkNETCoreSdkVersion>
     <_CoverletSdkNETCoreSdkVersion Condition="$(_CoverletSdkNETCoreSdkVersion.Contains('-'))">$(_CoverletSdkNETCoreSdkVersion.Split('-')[0])</_CoverletSdkNETCoreSdkVersion>
-    <_CoverletSdkMinVersionWithDependencyTarget>3.1.300</_CoverletSdkMinVersionWithDependencyTarget>
+    <_CoverletSdkMinVersionWithDependencyTarget>6.1.404</_CoverletSdkMinVersionWithDependencyTarget>
```

Comment:
Shouldn't this be `6.0.404`?
```suggestion
    <_CoverletSdkMinVersionWithDependencyTarget>6.0.404</_CoverletSdkMinVersionWithDependencyTarget>
```

Reply:
Correct  - this has to be 6.0.404. Thanks